fidimag/atomistic/llg.py

```diff
@@ -17,7 +18,7 @@ def __init__(self, mesh, name='unnamed', use_jac=False):
         *Arguments*
           name : the Simulation name (used for writing data files, for examples)
         """
-
+        self._micromagnetic = False
```

Comment:
Is it ok to leave the atomistic case as `not _micromagnetic` ? I'm just wondering if this will confuse other people, although It is just a name. Maybe we should discuss how to organise this in the next Finmag Friday